### DISCUSSION TITLE
Interesting find(maybe bug?) when using `@` inside codeblocks in Issue/Pull request description

### DISCUSSION THREAD

**Discussion by winwinashwin** [community/community#19041] [Projects and Issues]
When you type `@` inside codeblocks in Issue description/Pull request description of repositories, it brings up a list of users associated with the repo to tag. I understand the use of this feature in general, but is this intended inside codeblocks?

**Comment by rilysh** [comment]
Yes, if you saw a CODEOWNERS file you'd find that GitHub users are being mentioned though there. This is intended to be a feature.

**Comment by rilysh** [comment]
@nocoinman Sorry for ping, but would you please remove mark as answer from https://github.com/github-community/community/discussions/19041#discussioncomment-2994859
Actually it creates a highlight "1 discussion answered" which I don't like,I thought I can disable it but nope...sadly
